http/examples/get/client.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/apache/arrow/go/v15/arrow"
+	"github.com/apache/arrow/go/v15/arrow/ipc"
+	"github.com/apache/arrow/go/v15/arrow/memory"
+)
+
+func main() {
+	start := time.Now()
+	resp, err := http.Get("http://localhost:8000")
+	if err != nil {
+		panic(err)
+	}
+
+	if resp.StatusCode != http.StatusOK {
+		panic(fmt.Errorf("got non-200 status: %d", resp.StatusCode))
+	}
+	defer resp.Body.Close()
+
+	rdr, err := ipc.NewReader(resp.Body, ipc.WithAllocator(memory.DefaultAllocator))
+	if err != nil {
+		panic(err)
+	}
+	defer rdr.Release()
+
+	batches := make([]arrow.Record, 0)
+	defer func() {
+		for _, b := range batches {
+			b.Release()
+		}
+	}()
+
+	for rdr.Next() {
+		rec := rdr.Record()
+		rec.Retain()
+		batches = append(batches, rec)
+	}
+
+	if rdr.Err() != nil {
+		panic(rdr.Err())
+	}
+
+	execTime := time.Since(start)
+
+	fmt.Println(resp.ContentLength, " bytes recieved")
```

Comment:
When I use this with the Python example server, it prints `-1 bytes received` which means unknown. I believe this is because `resp.ContentLength` just gets the `Content-Length` header, which is not set. Is there a way to directly get the number of bytes in the response body (without making extra copies)?

Reply:
we can probably wrap the `resp.Body` with a simple counter pass through or a bufio object buffering the bytes allowing us to count them as we pass them to the `ipc` reader.

Reply:
Ok thanks; for now I'll just remove it for simplicity, since most of the other client examples do not print the size of received bytes (due to the added complication that would be needed).